language/move-lang/src/typing/translate.rs

```diff
@@ -622,8 +625,8 @@ fn check_phantom_params(
             }
         }
         Type_::Apply(_, n, ty_args) => match &n.value {
-            // Tuples cannot appear in structs, but we still report them as a non-phantom position in
-            // case they are ever allowed.
+            // Tuples cannot appear in structs, but we still report them as a non-phantom position
+            // for full information
```

Comment:
Super small nit-pick: end this sentence (and the ones in the comments below) with a period `.`, since that's how the old comments ended? _/me ducks_